spec/integration/item_displays_spec.rb

```diff
@@ -2,23 +2,11 @@
 
 describe 'mods_view', :type => :request do
   before :each do
-    @object = instantiate_fixture('druid_zt570tx3016', Dor::Item)
+    @current_user = admin_user # see spec_helper
+    @object = instantiate_fixture('zt570tx3016', Dor::Item)
     allow(Dor::Item).to receive(:find).and_return(@object)
-    @current_user = double(:webauth_user, :login => 'sunetid', :logged_in? => true, :privgroup => ADMIN_GROUPS.first, can_view_something?: true)
-    allow(@current_user).to receive(:is_admin).and_return(true)
-    allow(@current_user).to receive(:roles).and_return([])
-    allow(@current_user).to receive(:is_manager).and_return(false)
-    allow(@current_user).to receive(:permitted_apos).and_return([])
-
-    ##
-    # Stubbing of both ItemsController and CatalogController which are the
-    # controllers used in this spec, needing a valid return from current_user. A
-    # higher level up the inheritance chain stub of `ApplicationController` is
-    # insufficient here, due to a possible bug in rspec-mocks or our overuse of
-    # `allow_any_instance_of`.
-    allow_any_instance_of(ItemsController).to receive(:current_user).and_return(@current_user)
-    allow_any_instance_of(CatalogController).to receive(:current_user).and_return(@current_user)
   end
```

Comment:
Most of this `before` code is handled in the `admin_user` helper method.  There is no point in defining `is_manager`, `roles` or `permitted_apos` because the `is_admin` value overrides all other privileges, which can be clearly seen in the User model.